backend/kesaseteli/companies/models.py

```diff
@@ -1,10 +1,10 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
-from oidc.models import AbstractCompany
+from utils.models import UUIDModel
 
 
-class Company(AbstractCompany):
+class Company(UUIDModel):
```

Comment:
Hmn... I thought we will share the AbstractCompany model with all the fields below? Or this will be the next step after this PR

Reply:
I'll create another PR for that. Adding all the required stuff to shared.